setup.py

```diff
@@ -19,7 +19,7 @@
     install_requires=[
         'omero-py>=5.6.0',
         # TODO: Update to current release 0.7.*
-        'prometheus-client==0.2.*',
```

Comment:
Given the note above, what are the implications of uncapping this dependency?
In particular:
- are there some backwards-incompatible changes since version 0.2.x that would modify the usage of the library?
- as the upstream library is still sub-1.0.0, should it be capped to protect from future changes?

Reply:
I will try to see what happened when I re-introduced the cap, it might have been introduced when attempting to fix the initial issue